### ISSUE TITLE
Offset on drag-and-drop node creation

### ISSUE THREAD

**Issue by sonyaa** [CSE512-15S/fp-fiannaca-sonyaa#25]
Currently drag-and-dropped nodes are displayed with an offset. The reason for that is that the drag event (x,y) coordinates are calculated relative to the left side of the legend, but the (x,y) of rendered nodes are calculated relative to the left side of the working area.

**Comment by sonyaa** [comment]
@fiannaca thoughts?

**Comment by fiannaca** [comment]
Yeah, I saw this already, I just haven't got around to fixing it. I have some ideas, let me play with it for a couple minutes and see if I can fix it.

**Comment by fiannaca** [comment]
I've fixed it. I'm working on a couple of little additions to it. I'll push soon
